src/main/java/net/kodehawa/mantarobot/commands/currency/pets/HousePetType.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2016-2020 David Rubio Escares / Kodehawa
+ *
+ *  Mantaro is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *  Mantaro is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Mantaro.  If not, see http://www.gnu.org/licenses/
+ */
+
+package net.kodehawa.mantarobot.commands.currency.pets;
+
+import net.kodehawa.mantarobot.utils.Utils;
+import net.kodehawa.mantarobot.utils.commands.EmoteReference;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum HousePetType {
+    DOG(EmoteReference.DOG, "Dog", List.of(HousePetAbility.CATCH, HousePetAbility.CHEER), 40000, 200, true),
+    CAT(EmoteReference.CAT, "Cat", List.of(HousePetAbility.FISH, HousePetAbility.CHEER), 30000, 150, true),
+    RAT(EmoteReference.HAMSTER, "Hamster", List.of(HousePetAbility.CHEER), 5000, 30, true),
```

Comment:
CHOP + CHEER instead of just CHEER for consistency?

Reply:
yup, done